pkg/redfish/redfish.go

```diff
@@ -65,6 +65,7 @@ type GenericCollection struct {
 	Otype         string       `json:"@odata.type"`
 	Description   string       `json:"Description"`
 	Members       []ResourceID `json:"Members"`
+	Outlets       []ResourceID `json:"Outlets"`             // For HPE PDU Outlets
```

Comment:
A different struct should be created for this

Reply:
The outlet slice is the same as the members slice.  The PDU folks decided to use "Outlets" instead of "Members" in their redfish.  If Outlets returns with items, they are copied to Members.

Reply:
@mbuchmann-hpe Ok. You can keep it as is then.
@nieuwsma Since this struct is never in output for HSM, no swagger needs updating.